ksqldb-common/src/main/java/io/confluent/ksql/schema/Operator.java

```diff
@@ -61,6 +65,10 @@ public String getSymbol() {
    * @return the result schema.
    */
   public SqlType resultType(final SqlType left, final SqlType right) {
+    if (left == null || right == null) {
```

Comment:
nit: let's extract this into a single `checkForNull` method and reuse it above (makes sure the error messages are identical)